src/pages/caseDetail/caseDetailView.js

```diff
@@ -53,6 +55,19 @@ const options = (classes) => ({
 
 // Main case detail component
 const CaseDetail = ({ data, classes }) => {
+  const dispatch = useDispatch();
```

Comment:
pageSpecificStatsController.js will take care of this dispatch. As we are using dashboard data only for the stats